Backend/Backend/Services/FileService.cs

```diff
@@ -90,30 +77,18 @@ public bool Delete(string path)
 
         }
        
-        public async Task<bool?> DeleteFile(string id)
+        public bool? DeleteFile(string path)
         {
-            Guid? guid = Guid.Parse(id);
-
-
-            Files? file = await  _db.File!.FirstOrDefaultAsync(u => u.FileId == guid);
-            if (file != null)
+            var fullPath = GetPath(path);
+            if (File.Exists(fullPath))
```

Comment:
why remove the exceptions from here? This is the part of the code that is responsible for it.